src/main/java/org/concordion/api/CommandCall.java

```diff
@@ -90,4 +91,18 @@ public String getParameter(String camelCaseParameterName, String spinalCaseParam
         }
         return null;
     }
+
+    public boolean hasNonExampleChildren() {

```

Comment:
This method is now doing more than just checking for non-example children. My preference would be to keep the logic just checking for non-example children, and add an additional method to the caller to check whether the command should execute for non-example children.